### PR TITLE
new sound effects for dropping, throwing, picking up, and equipping certain things!

### PR DESCRIPTION
**Pull request by UselessTheremin**
ports https://github.com/tgstation/tgstation/pull/46563
ports https://github.com/tgstation/tgstation/pull/46387
ports https://github.com/tgstation/tgstation/pull/46677
ports https://github.com/tgstation/tgstation/pull/53392

essentially, there are new sound effects for picking up, throwing, dropping, and equipping some items.
video included:
https://streamable.com/koz4r

NOTE: other players can hear these sounds, but the sounds are blocked by walls and aren't that loud to begin with.
:cl:  
soundadd: new specific sounds for equipping clothing, picking up items, throwing items, and dropping items 
/:cl:

### CODE REVIEW

**Comment by UselessTheremin** [comment]
coommit wall moment

File: code/game/objects/items.dm [hunk 1]
```diff
@@ -420,6 +431,11 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 		if(item_action_slot_check(slot, user)) //some items only give their actions buttons when in a specific slot.
 			A.Grant(user)
 	item_flags |= IN_INVENTORY
+	if(!initial)
+	if(equip_sound && !initial &&(slot_flags & slotdefine2slotbit(slot)))
+			playsound(src, equip_sound, EQUIP_SOUND_VOLUME, TRUE, ignore_walls = FALSE)
+		else if(slot == SLOT_HANDS)
+			playsound(src, pickup_sound, PICKUP_SOUND_VOLUME, ignore_walls = FALSE)
```
Comment (Absolucy) [inline code/game/objects/items.dm]:
```suggestion
	if(!initial)
		if(equip_sound && !initial &&(slot_flags & slotdefine2slotbit(slot)))
			playsound(src, equip_sound, EQUIP_SOUND_VOLUME, TRUE, ignore_walls = FALSE)
		else if(slot == SLOT_HANDS)
			playsound(src, pickup_sound, PICKUP_SOUND_VOLUME, ignore_walls = FALSE)
```

**Review by Absolucy — CHANGES_REQUESTED** [review]

**Comment by UselessTheremin** [comment]
FUCK

**Comment by JamieD1** [comment]
Pretty cool, how loud are they compared to other things.

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**